gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -630,12 +629,6 @@ public float getScaleY () {
 	/** Returns the color of this sprite. If the returned instance is manipulated, {@link #setColor(Color)} must be called
 	 * afterward. */
 	public Color getColor () {
-		int intBits = NumberUtils.floatToIntColor(vertices[C1]);
-		Color color = this.color;
-		color.r = (intBits & 0xff) / 255f;
-		color.g = ((intBits >>> 8) & 0xff) / 255f;
-		color.b = ((intBits >>> 16) & 0xff) / 255f;
-		color.a = ((intBits >>> 24) & 0xff) / 255f;
```

Comment:
I definitely like this removal; the scaling NathanSweet was referring to happens here and in floatToIntColor().

NumberUtils.floatToIntColor() does have some oddities because it tries to create an 8-bit alpha from a 7-bit value. My preference when doing that is to copy the MSB of alpha into the previously-vacant LSB of alpha, like:
```java
    public static int floatToIntColor (float value) {
        int intBits = Float.floatToRawIntBits(value);
        intBits |= (intBits >>> 31) << 24;
        return intBits;
    }
```

This also avoids some float math.